README.md

```diff
@@ -29,23 +33,27 @@ The intended usage is as an npm script:
 {
   ...
   "scripts": {
-    "find-new-rules": "eslint-find-new-rules path/to/eslint-config"
+    "eslint-find-option-rules": "eslint-rule-finder --option path/to/eslint-config"
```

Comment:
`--option` looks like an actual option to specify as argument to me.
Personally I'd prefer if this could be an example with an actual command and later on (L42-45) we could simply state `eslint-rule-finder [option] <file>` or sth. like that (possibly just the way the ESLint cli does it).